packages/ra-ui-materialui/src/input/FileInput.tsx

```diff
@@ -195,10 +192,9 @@ export const FileInput = (props: FileInputProps) => {
                                 onRemove={onRemove(file)}
                                 className={FileInputClasses.removeButton}
                             >
-                                {cloneElement(childrenElement as ReactElement, {
-                                    record: file,
-                                    className: FileInputClasses.preview,
```

Comment:
With this change, you are removing the use of `FileInputClasses.preview` with is still in the docs.

Reply:
Thanks for your remark. We will remove it from the docs in an other PR.